cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/builder/Fluent.kt

```diff
@@ -63,6 +69,8 @@ fun LanguageFrontend<*, *>.translationResult(
         executePassesSequentially(ctx, node, mutableSetOf())
     }
 
+    node.inferrPseudolocations()
```

Comment:
```suggestion
    node.inferPseudolocations()
```
What does this do? Do we need it here as well? There should only be one `TranslationResult`